src/parser/jobs/mnk/modules/Forms.tsx

```diff
@@ -115,41 +112,30 @@ export default class Forms extends Module {
 	}
 
 	private onComplete(): void {
-		// Wasting PB
-		if (this.poorForms >= 1) {
-			this.suggestions.add(new Suggestion({
-				icon: ACTIONS.PERFECT_BALANCE.icon,
-				severity: SEVERITY.MAJOR,
-				content: <Trans id="mnk.forms.suggestions.perfectbalance.content">
-					Avoid using <ActionLink {...ACTIONS.FORM_SHIFT}/> during <ActionLink {...STATUSES.PERFECT_BALANCE}/>. It does nothing and takes up a GCD you could better use for doing damage.
-				</Trans>,
-				why: <Trans id="mnk.forms.suggestions.perfectbalance.why">
-					<Plural value={this.poorForms} one="GCD was" other="GCDs were" /> wasted under <StatusLink {...STATUSES.PERFECT_BALANCE}/> by <ActionLink {...ACTIONS.FORM_SHIFT}/>.
-				</Trans>,
-			}))
-		}
-
 		// Using the wrong form
-		if (this.formless >= 1) {
-			this.suggestions.add(new Suggestion({
-				icon: ACTIONS.FORM_SHIFT.icon,
-				severity: SEVERITY.MAJOR,
-				content: <Trans id="mnk.forms.suggestions.formless.content">
-					Avoid using <ActionLink {...ACTIONS.DRAGON_KICK}/> and <ActionLink {...ACTIONS.ARM_OF_THE_DESTROYER}/> outside of <StatusLink {...STATUSES.OPO_OPO_FORM}/>. Their special effects only activate when in the correct form and <ActionLink {...ACTIONS.BOOTSHINE} /> has equal or higher potency depending on crits.
-				</Trans>,
-				why: <Trans id="mnk.forms.suggestions.formless.why">
-					<Plural value={this.formless} one="# combo-starter was" other="# combo-starters were" />  used Formlessly, cancelling this special effects.
-				</Trans>,
-			}))
-		}
+		this.suggestions.add(new TieredSuggestion({
+			icon: ACTIONS.FORM_SHIFT.icon,
+			severity: SEVERITY.MEDIUM,
```

Comment:
I don't think severity should be specified as this is TieredSuggestion; it gets overwritten by `tiers`
Anyway medium is the default severity :p Sorry for the nitpick

Reply:
It's a leftover, good catch!